conf.d/win32_event_log.yaml.example

```diff
@@ -1,4 +1,9 @@
 init_config:
+    # The (optional) tag_event_id setting will add an event id tag to each
+    # event sent from this check. Defaults to false.
+    # tag_event_id: true
+
+    tag_event_id: false
```

Comment:
I think we kind of have an implicit convention that the default parameter should be commented, and nothing outside the comment, i.e. : 
```
 init_config:
    # The (optional) tag_event_id setting will add an event id tag to each
    # event sent from this check. Defaults to false.
    # tag_event_id: false
```